packages/ui/src/kit/index.js

```diff
@@ -33,3 +33,4 @@ export { default as VModalCard } from '@/kit/VModal/VModalCard';
 export { default as VModalInfoCard } from '@/kit/VModal/VModalInfoCard';
 export { default as VProgressCircle } from '@/kit/VProgressCircle';
 export { default as VTag } from '@/kit/VTag';
+export { default as VAccordion } from '@/kit/VAccordion';
```

Comment:
do we need item too? @dudiq

Reply:
@kazurus did add it

Reply:
i think, no needed. because we always using full relative path to component, not the whole ui-kit